hey.go

```diff
@@ -89,6 +92,7 @@ Options:
   -t  Timeout in seconds. Default is 20, use 0 to disable.
   -A  HTTP Accept header.
   -d  HTTP request body.
+  -D  HTTP request body from file, for example, -D /root/reqbody, -D ../reqbody
```

Comment:
Can we make the sample path more obvious?

-D HTTP request body from file, for example, /home/user/file.txt or ./file.txt